Documentation/Images/AutomaticScreenshots/WebList/WithExportButtons.rst.txt

```diff
@@ -0,0 +1,6 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. figure:: /Images/AutomaticScreenshots/WebList/WithExportButtons.png
+   :class: with-shadow
+
+   The list module with export buttons after activating the single-table mode
```

Comment:
The screenshots should be updated yet again ;) the button is called "download" now I think. See https://review.typo3.org/c/Packages/TYPO3.CMS/+/69794